resources/tests.mjs

```diff
@@ -4,39 +4,29 @@ const numberOfItemsToAdd = 100;
 export const Suites = [];
 const ENTER_KEY_CODE = 13;
 
-let triggerEnter = function (element, type) {
-    const event = document.createEvent('Events');
-    event.initEvent(type, true, true);
-    event.keyCode = ENTER_KEY_CODE;
-    event.which = ENTER_KEY_CODE;
-    event.key = 'ENTER';
-    element.dispatchEvent(event);
-}
-
 Suites.push({
     name: 'VanillaJS-TodoMVC',
     url: 'todomvc/vanilla-examples/vanillajs/index.html',
-    prepare(runner, contentWindow, contentDocument) {
-        return runner.waitForElement('.new-todo').then(element => {
-            element.focus();
-            return element;
-        });
+    async prepare(page) {
+        const element = await page.waitForElement('.new-todo');
```

Comment:
This local variable seems unnecessary.
We can just do: `(await ~).focus();`

Reply:
DONE.